src/tests/test_pyodide.py

```diff
@@ -246,7 +246,8 @@ def test_keyboard_interrupt(selenium):
                         triggerKeyboardInterrupt()
             `)
         } catch(e){}
-        return pyodide.runPython('x')
+        pyodide.setInterruptBuffer(undefined);
```

Comment:
Is it necessary to set the buffer back to `undefined` ?

Reply:
I think it's a good idea because `setInterruptBuffer` changes global state so if we don't do this and having an interrupt buffer changes the behavior of other tests it could cause confusion.